pkg/server/container_create_windows.go

```diff
@@ -169,7 +169,7 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 		return nil, errors.Wrapf(err, "failed to generate container %q spec", id)
 	}
 
-	log.G(ctx).Debugf("Container %q spec: %#+v", id, spew.NewFormatter(spec))
+	log.G(ctx).Debugf("Container %q, runtime handler: %q spec: %#+v", id, sandbox.RuntimeHandler, spew.NewFormatter(spec))
```

Comment:
I think it would be preferred to move all of these extra fields out of the log message:
```
log.G(ctx).WithFields(logrus.Fields{
    "id": id,
    "runtimeHandler": sandbox.RuntimeHandler,
    "spec": spew.NewFormatter(spec),
}).Debug("Container creation")
```